instagram/src/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './PostContainer.css';
+import CommentSection from '../CommentSection/CommentSection';
+import styled from 'styled-components';
+
+
+const PostContainer = (props) => {
+        return (
+            <PostContainerStyle>
+                <UsernameHeader>
+                    <UsernameImage src={props.obj.thumbnailUrl} alt='user thumbnail' />
+                    <UsernameText>{props.obj.username}</UsernameText>
+                </UsernameHeader>
+                <PostImage className="postImage" src={props.obj.imageUrl} alt='' />
+                <CommentSection
+                    key={props.obj.timestamp}
+                    obj={props.obj}
+                />
+            </PostContainerStyle>
+        )
+}
+
+const PostContainerStyle = styled.div`
+    width: 80%;
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+`;
+
+const UsernameHeader = styled.div`
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    margin-bottom: 10px;
+`;
+
```

Comment:
If you are using Styled Components for all of your components and files I would suggest putting all of them in a separate file and then importing them in.